### PR TITLE
DAM: Fix/set cache-control headers

### PR DESCRIPTION
**Pull request by dkarnutsch**
## Example

-   [x] I have verified if my change requires an example

## Changeset

-   [x] I have verified if my change requires a changeset

### CODE REVIEW

File: packages/api/cms-api/src/dam/images/images.controller.ts [hunk 1]
```diff
@@ -107,7 +107,9 @@ export class ImagesController {
             throw new NotFoundException();
         }
 
-        return this.getCroppedImage(file, params, accept, res);
+        return this.getCroppedImage(file, params, accept, res, {
+            "cache-control": "max-age=3600, public", // Public cache (1 hour)
```
Comment (dkarnutsch) [inline packages/api/cms-api/src/dam/images/images.controller.ts:111]: As discussed: 1 day

File: packages/api/cms-api/src/dam/files/files.controller.ts [hunk 2]
```diff
@@ -147,7 +147,7 @@ export function createFilesController({ Scope: PassedScope }: { Scope?: Type<Dam
             }
 
             res.setHeader("Content-Disposition", "attachment");
-            return this.streamFile(file, res, { range });
+            return this.streamFile(file, res, { range, overrideHeaders: { "cache-control": "max-age=86400, public" } }); // Public cache (1 hour)
```
Comment (thomasdax98) [inline packages/api/cms-api/src/dam/files/files.controller.ts:150]: If I'm not mistaken:

```suggestion
            return this.streamFile(file, res, { range, overrideHeaders: { "cache-control": "max-age=86400, public" } }); // Public cache (1 day)
```

File: packages/api/cms-api/src/dam/files/files.controller.ts [hunk 3]
```diff
@@ -163,7 +163,7 @@ export function createFilesController({ Scope: PassedScope }: { Scope?: Type<Dam
                 throw new NotFoundException();
             }
 
-            return this.streamFile(file, res, { range });
+            return this.streamFile(file, res, { range, overrideHeaders: { "cache-control": "max-age=86400, public" } }); // Public cache (1 hour)
```
Comment (thomasdax98) [inline packages/api/cms-api/src/dam/files/files.controller.ts:166]:
```suggestion
            return this.streamFile(file, res, { range, overrideHeaders: { "cache-control": "max-age=86400, public" } }); // Public cache (1 day)
```

File: packages/api/cms-api/src/dam/images/images.controller.ts [hunk 4]
```diff
@@ -123,7 +125,9 @@ export class ImagesController {
             throw new NotFoundException();
         }
 
-        return this.getCroppedImage(file, params, accept, res);
+        return this.getCroppedImage(file, params, accept, res, {
+            "cache-control": "max-age=86400, public", // Public cache (1 hour)
```
Comment (thomasdax98) [inline packages/api/cms-api/src/dam/images/images.controller.ts:129]:
```suggestion
            "cache-control": "max-age=86400, public", // Public cache (1 day)
```

File: packages/api/cms-api/src/dam/files/files.controller.ts (hunk 3 above)
Comment (dkarnutsch) [reply packages/api/cms-api/src/dam/files/files.controller.ts]: yes, thanks 🤦 

a4ff03f0033b33f8478730b618908bd9a3319cb6

**Review by johnnyomair — APPROVED** [review]

**Review by thomasdax98 — APPROVED** [review]

**Pull request merged**